### ISSUE TITLE
Hanging line in multi-controller toplogy print

### ISSUE THREAD

**Issue by finikorg** [intel/thunderbolt-software-user-space#48]
In a system with several Thunderbolt controllers (like Macbook Pro for example) indentation is broken

```
Controller 0
│   ├─ Details:
│   │   ├─ Name: Macintosh, Apple Inc.
│   │   └─ Security level: SL0 (none)
│   │
│   └─ Thunderbolt to FireWire Adapter, Apple, Inc.
│       └─ Details:
│           ├─ Route-string: 0-3
│           ├─ Authorized: Yes
│           ├─ In ACL: No
│           └─ UUID: e0376f00-0300-0100-ffff-ffffffffffff
Controller 1
│     └─ Details:
│         ├─ Name: Macintosh, Apple Inc.
│         └─ Security level: SL0 (none)
```

**Comment by ybernat** [comment]
Are you talking about the details of controller 1? Because this will happen for controller 0 too if nothing is connected, probably.
I think this issue could be improved if the "expected output" would be mentioned too.

**Comment by finikorg** [comment]
I fixed this output in the PR #46 
https://travis-ci.org/finikorg/thunderbolt-software-user-space/builds/327175719
The expected output is:
```
Controller 0
│   ├─ Details:
│   │   ├─ Name: Controller, Mock Device
│   │   └─ Security level: SL2 (secure)
│   │
│   └─ Thunderbolt Cable, Mock Vendor
│       └─ Details:
│           ├─ Route-string: 0-1
│           ├─ Authorized: No
│           ├─ In ACL: No
│           └─ UUID: b564f8bd-ef28-48ad-89c5-c1fc603d902f
Controller 1
    ├─ Details:
    │   ├─ Name: Controller, Mock Device
    │   └─ Security level: SL0 (none)
    │
    └─ Thunderbolt Cable, Mock Vendor
        └─ Details:
            ├─ Route-string: 1-1
            ├─ Authorized: Yes
            ├─ In ACL: No
            └─ UUID: 3510542a-afa4-4c02-b720-ad6e618796a1
```
